lib/src/model/settings/board_preferences.dart

```diff
@@ -137,6 +148,23 @@ class BoardPrefs with _$BoardPrefs {
       pieceAnimation ? const Duration(milliseconds: 150) : Duration.zero;
 }
 
+enum ShapeColor {
+  green,
+  red,
+  blue,
+  cyan,
+  purple;
+
+  Color get color => switch (this) {
+        ShapeColor.green => LichessColors.green,
```

Comment:
Are these the correct colors to use here? Or should we rather use the colors from here: https://github.com/lichess-org/chessground/blob/54a7e71bf88701c1109d3b9b8106b464012b94cf/src/state.ts#L178

Reply:
I'd say let's use the colors defined in lichess website indeed.

Also I think showing the name of the color in the picker is not enough. Could you also show a coloured square so the user can look at the color? Thanks.

Reply:
> I'd say let's use the colors defined in lichess website indeed.
👍 

> Also I think showing the name of the color in the picker is not enough. Could you also show a coloured square so the user can look at the color? Thanks.

Ah good idea! Maybe we could even show a small arrow instead of a square?